tools/prep/prepare_space_xml.py

```diff
@@ -316,6 +316,7 @@ def main():
                              "solvers (StarExec IDs, wrapped IDs if -w)")
     parser.add_argument("--include-non-competitive",
                         default=False,
+                        dest="run_noncompetitive",
```

Comment:
why add this?

Reply:
I just find it nicer to write and remember.  Do you prefer the include_non_competitive?  Or did you mean this question more generally?